src/restricted-endpoints/restricted-endpoints.module.ts

```diff
@@ -24,43 +19,14 @@ export class RestrictedEndpointsModule implements NestModule {
   static proxy: RequestHandler;
 
   configure(consumer: MiddlewareConsumer): any {
-    this.applyProxyForPermissionlessCouchdbEndpoints(consumer);
     consumer
       .apply(
         json({ limit: '10mb' }),
         urlencoded({ extended: true, limit: '10mb' }),
       )
       .forRoutes('*');
-  }
-
-  private applyProxyForPermissionlessCouchdbEndpoints(
-    consumer: MiddlewareConsumer,
-  ) {
+    consumer.apply(CombinedAuthGuard).exclude('_session');
     this.initializeProxy();
```

Comment:
do we still need `initializeProxy` at all? RestrictedEndpointsModule.proxy is not applied anymore, so it seems to be unused?

Reply:
Its still used for the file upload but I can move it there